build_msvc/msvc-autogen.py

```diff
@@ -42,8 +43,26 @@ def parse_makefile(makefile):
                     lib_sources[current_lib] = []
                     break
 
+def set_custom_paths(qtDir, protocPath):
+    with open('build_msvc\\Directory.build.props', 'r', encoding='utf-8') as rfile:
```

Comment:
I would prefer to use absolute path here because people may double click the script on their computer instead of running it in `cmd`, maybe you could add `MSVC_DIR` right after `SOURCE_DIR`
```python3
MSVC_DIR =  os.path.abspath(os.path.dirname(__file__))
```
and use
```python3
with open(os.path.join(MSVC_DIR, 'Directory.Build.props'), 'r', encoding='utf-8') as rfile:
```